flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -1807,6 +1808,7 @@ private void releaseJobResources(JobID jobId, Exception cause) {
                         });
         taskManagerMetricGroup.removeJobMetricsGroup(jobId);
         changelogStoragesManager.releaseStateChangelogStorageForJob(jobId);
+        changelogStoragesManager.releaseStateChangelogStorageViewForJob(jobId);
```

Comment:
Should these two methods be combined into a single one, e.g. `releaseResourcesForJob` to abstract from the details?